packages/translations/src/i18next/i18next.js

```diff
@@ -29,7 +29,7 @@ const getUrlBase = (path = '') => {
 };
 
 // TODO: [app-link-refactor] - Remove backwards compatibility for `deriv.app`
-const isStaging = () => /staging\.deriv\.app|staging-app\.deriv\.com/i.test(window.location.hostname);
+const isStaging = () => true;
```

Comment:
Was this committed by mistake?  If not can we remove this line and return `true` where this function is invoked?

Reply:
Yes thanks I was double checking if translations were working for the form messages 👍